internal/goglfw/context_windows.go

```diff
@@ -587,3 +586,22 @@ func ExtensionSupported(extension string) (bool, error) {
 	// Check if extension is in the platform-specific string
 	return window.context.extensionSupported(extension), nil
 }
+
+// copied from golang.org/x/sys/windows/syscall.go
```

Comment:
Start with `bytePtrToString...` for this comment. `bytePtrToString converts a pointer to a byte to a string. This is copied...` maybe?